src/webgpu/web_platform/copyToTexture/ImageBitmap.spec.ts

```diff
@@ -179,15 +181,79 @@ got [${failedByteActualValues.join(', ')}]`;
 
     return entry.get(color)!;
   }
+
+  // Helper functions to generate imagePixels based input configs.
+  getImagePixels(
+    format: UncompressedTextureFormat,
+    width: number,
+    height: number,
+    transparentOp: TransparentOp,
+    orientationOp: OrientationOp
```

Comment:
totally optional: I noticed that in the usages of this method, you have comments about which argument is which. Instead of that, you could use a dictionary to have named arguments, e.g. `getImagePixels({ format, width, height, transparentOp }: { format: UncompressedTextureFormat, width: number, height: number, transparentOp: TransparentOp, orientationOp: OrientationOp })`

Reply:
Glad to have it. Done.